cms/models_test.py

```diff
@@ -265,3 +270,64 @@ def test_home_catalog():
     assert not home.catalog
     catalog = CatalogGridSectionFactory(parent=home)
     assert home.catalog == catalog
+
+
+def test_certificate_for_bootcamp_run_page():
+    """
+    The Certificate property should return expected values if associated with a CertificatePage
+    """
+    bootcamp_run_page = BootcampRunPageFactory.create()
+    assert models.CertificatePage.can_create_at(bootcamp_run_page)
+    assert not models.SignatoryPage.can_create_at(bootcamp_run_page)
+
+    certificate_page = CertificatePageFactory.create(
+        parent=bootcamp_run_page,
+        bootcamp_run_name="bootcamp_run",
+        certificate_name="certificate_name",
+        location="location",
+        signatories__0__signatory__name="Name",
+        signatories__0__signatory__title_1="Title_1",
+        signatories__0__signatory__title_2="Title_2",
+        signatories__0__signatory__organization="Organization",
+        signatories__0__signatory__signature_image__title="Image",
```

Comment:
shouldn't we use SignatoryPageFactory here?

Reply:
See how it's being done in xPRO, you will get the idea of it.